### PR TITLE
[MIG] crm_claim: Migration to 14.0

### PR DESCRIPTION
**Pull request by redapureskill**
Migrating crm_claim from V13 to **V14**:
=============================================
#TODO Change version in __manifest__.py file  :  Done.
#TODO Migration process file crm_claim.data : {
	crm_claim_data.xml : Done.
} : Done.
#TODO Migration process file crm_claim.demo : {
	crm_claim_demo.xml : Done.
} : Done.
#TODO Migration process file crm_claim.models : {
	crm_claim.py           : Done.
	crm_claim_category.py  : Done. 
	crm_claim_stage.py     : Done.
	res_partner.py         : Done.

} : Done
#TODO Migration process file crm_claim.report : {
	crm_claim_report.py       : Done.
	crm_claim_report_view.xml : Done.
}: Done.
#TODO Migration process file crm_claim.tests : Done.
#TODO Migration process file crm_claim.report :{
	crm_claim_view.xml :          Done.
	crm_claim_category_view.xml : Done.
	crm_claim_stage_view.xml :    Done.
	res_partner_view.xml :        Done.
}: Done.

### CODE REVIEW

File: crm_claim/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2015-2017 Odoo S.A.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2018 Tecnativa - Cristina Martin R.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Claims Management",
+    "version": "14.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "author": "Odoo S.A., " "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (skeller1) [inline crm_claim/__manifest__.py]: `Odoo S.A., " "Tecnativa, " "Odoo Community Association (OCA)` => `Odoo S.A., Tecnativa, Odoo Community`

File: crm_claim/README.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to new menu **CRM > Configuration > Claim > Categories** and create as
```
Comment (skeller1) [inline crm_claim/README.rst:44]: maybe we could add the information that we have to do this in debug mode

File: crm_claim/views/crm_claim_category_views.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <field name="name">Claim Categories</field>
+        <field name="res_model">crm.claim.category</field>
+        <field name="view_id" ref="crm_claim_category_tree" />
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Click to create a claim category.
+            </p>
+            <p>
+            Create claim categories to better manage and classify your
+            claims. Some example of claims can be: preventive action,
```
Comment (skeller1) [inline crm_claim/views/crm_claim_category_views.xml]: Maybe i am wrong: examples instead of example

**Review by skeller1 — APPROVED** [review]
:+1: LGTM code and functional review locally (runbot is not working at the moment)

Maybe one wish: adding `ondelete="restrict"` to stage_id and categ_id fields in crm.claim model to prevent information loss. but this is not blocking.

Thank you for supporting OCA

**Comment by Reyes4711-S73** [comment]
Functional review

File: crm_claim/views/crm_claim_views.xml [hunk 4]
```diff
@@ -13,48 +13,48 @@
                 <field name="activity_date_deadline" />
                 <field name="activity_summary" />
                 <field name="categ_id" string="Type" />
-                <field name="date_deadline" attrs="{'invisible': [(1, '=', 1)]}" />
-                <field name="date_closed" attrs="{'invisible': [(1, '=', 1)]}" />
-                <field name="company_id" groups="base.group_multi_company" />
+                <field attrs="{'invisible': [(1, '=', 1)]}" name="date_deadline" />
```
Comment (dsolanki-initos) [inline crm_claim/views/crm_claim_views.xml:16]: Why we need to change this?

File: crm_claim/views/crm_claim_views.xml [hunk 5]
```diff
@@ -13,48 +13,48 @@
...
                         options="{'clickable': 1}"
+                        widget="statusbar"
                     />
                 </header>
                 <sheet string="Claims">
                     <group>
                         <field name="name" />
                         <field name="date" />
-                        <field name="company_id" groups="base.group_multi_company" />
+                        <field groups="base.group_multi_company" name="company_id" />
```
Comment (dsolanki-initos) [inline crm_claim/views/crm_claim_views.xml:38]: Is it mentioned somewhere to use groups first and then name?

File: crm_claim/__manifest__.py [hunk 6]
```diff
@@ -7,7 +7,7 @@
     "name": "Claims Management",
     "version": "14.0.1.0.0",
     "category": "Customer Relationship Management",
-    "author": "Odoo S.A., Tecnativa, Odoo Community",
+    "author": "Odoo Community Association (OCA)",
```
Comment (dsolanki-initos) [inline crm_claim/__manifest__.py:10]: Why authors are removed?

**Review by Reyes4711-S73 — APPROVED** [review]

**Review by claudiagn — APPROVED** [review]
functional review ok! LGTM

**Review by simahawk — CHANGES_REQUESTED** [review]
as per https://github.com/OCA/crm/pull/368 fix the build first :pray:

**Comment by anajuaristi** [comment]
Hello
@simahawk  #368 is related to crm_lead and this is crm_claim, so they are different objects. Am I missing something?

On the other side, I tested functionally crm_claim on V14 and  LGTM :+1:

**Comment by simahawk** [comment]
> Hello
> @simahawk #368 is related to crm_lead and this is crm_claim, so they are different objects. Am I missing something?
> 
> On the other side, I tested functionally crm_claim on V14 and LGTM +1

Ciao Ana! :)

I know but the point is that nobody will be able to merge  PR till that issue is gone.

**Comment by oihane** [comment]
> > Hello
> > @simahawk #368 is related to crm_lead and this is crm_claim, so they are different objects. Am I missing something?
> > On the other side, I tested functionally crm_claim on V14 and LGTM +1
> 
> Ciao Ana! :)
> 
> I know but the point is that nobody will be able to merge PR till that issue is gone.

fix done in #388

**Comment by redapureskill** [comment]
> > > Hello
> > > @simahawk #368 is related to crm_lead and this is crm_claim, so they are different objects. Am I missing something?
> > > On the other side, I tested functionally crm_claim on V14 and LGTM +1
> > 
> > 
> > Ciao Ana! :)
> > I know but the point is that nobody will be able to merge PR till that issue is gone.
> 
> fix done in #388

still some checks not successfull please check #388

**Comment by Daniel-CA** [comment]
@redapureskill You should check too @dsolanki-initos stated

**Comment by oihane** [comment]
@redapureskill you should rebase, even there was a check in red, was a minor problem because there was no report to compare with.

**Comment by anajuaristi** [comment]
Hello, is important you make rebase to Travis be able to pass.
It would be great if we could merge this ASAP

**Comment by dsolanki-initos** [comment]
It would be great if this PR is merged.

**Comment by simahawk** [comment]
@dsolanki-initos @redapureskill someone has to rebase it :wink:

File: crm_claim/views/crm_claim_views.xml (hunk 4 above)
Comment (oihane) [reply crm_claim/views/crm_claim_views.xml:16]: And I don't think it works properly. It shows only header, without data, it does not make any sense.

**Comment by dsolanki-initos** [comment]
Hi @redapureskill  it would be great if you could rebase it so that this PR can be merged .

**Comment by dsolanki-initos** [comment]
Hi @redapureskill is it possible to rebase? we need to merge it ASAP.

**Comment by dsolanki-initos** [comment]
Hi @redapureskill can you please rebase as we need to merge it ASAP.

**Comment by anajuaristi** [comment]
@dsolanki-initos @simahawk @oihane @Daniel-CA 

Hi tech people. I think @redapureskill is not reading the messages or he is too busy to go on with this. I propose to make a manual merge ourselves.
Please comment if it's possible and if you consider it would be right if we do it.
Thank you!
Ana

**Comment by Daniel-CA** [comment]
I have done another PR with last fixes and updated. #391

**Comment by redapureskill** [comment]
> Hi @redapureskill can you please rebase as we need to merge it ASAP.

fixed and rebased, sorry mom passed away!

**Comment by redapureskill** [comment]
LGTM

**Comment by dsolanki-initos** [comment]
Seems that PR is Ready to merge It would be great if it could merged

File: crm_claim/__manifest__.py (hunk 6 above)
Comment (NL66278) [reply crm_claim/__manifest__.py:10]: @redapureskill Please restore Odoo SA and Tecnativa as authors. Then I could merge this.

Comment (Daniel-CA) [reply crm_claim/__manifest__.py:10]: @NL66278 This changes among others are done in this #391

**Comment by pedrobaeza** [comment]
Superseded by #391

File: crm_claim/__manifest__.py (hunk 6 above)
Comment (NL66278) [reply crm_claim/__manifest__.py:10]: @Daniel-CA I do not quite understand what you mean with the reference to the other PR. We should not just delete authors...

Comment (pedrobaeza) [reply crm_claim/__manifest__.py:10]: Please review the other PR as superseding this one.

**Pull request closed without merging**